Anagram.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Anagram {
+    public static void main(String[] args) {
+        Scanner scanner=new Scanner(System.in);
+        String stringOne=scanner.nextLine();
+        String stringTwo=scanner.nextLine();
+
+        String[] arrayOne=stringOne.split("[^a-zA-Z]");
+        String[] arrayTwo=stringTwo.split("[^a-zA-Z]");
+        stringOne="";
+        stringTwo="";
+
+        for(int i=0;i<arrayOne.length;i++){
+            stringOne+=arrayOne[i];
+        }
+
+        for(int i=0;i<arrayTwo.length;i++){
+            stringTwo+=arrayTwo[i];
+        }
+
```

Comment:
instead of this, you can directly use the replaceAll() function a = a.replaceAll("\\s","");

Reply:
> instead of this, you can directly use the replaceAll() function a = a.replaceAll("\s","");

thanks sir. Ssurely Would work on it .